src/kibana/components/config/defaults.js

```diff
@@ -43,6 +43,21 @@ define(function (require) {
       value: 6,
       description: 'The maximum geoHash size allowed in a tile map',
     },
+    'visualization:tileMap:wms': {
+      value: false,
+      description: 'Set for WMS mapping provider'
```

Comment:
@rashidkpc I tried, but HTML tags are shown as is rather than parsed. Any tips on fixing this, my angular knowledge is slim:(